### DISCUSSION TITLE
sync indicator in the addon bar

### DISCUSSION THREAD

**Discussion by misterhsp** [floccusaddon/floccus#1082] [General]
Since the last update I no longer see a sync indicator in the addon bar (blue rectangle). Is there something broken or has this been removed? If that has been removed, I would like to have that back.

...

**Comment by marcelklehr** [comment]
The blue rectangle has been replaced with a change in the whole icon:
![Screenshot from 2022-03-19 12-58-29](https://user-images.githubusercontent.com/986878/159120193-7291b2bd-847b-4c12-a4c5-ad22ec72e818.png)
![Screenshot from 2022-03-19 12-59-06](https://user-images.githubusercontent.com/986878/159120200-80a6585a-e7ae-4f74-bdcf-e14f8595e91e.png)

**Reply by misterhsp** [reply]
I thought it was better before. I am slightly visually impaired and can only see it badly.

...

**Comment by misterhsp** [comment]
Maybe you could change the color of the icon during the sync process. If the sync fails, it also turns red.

...

**Reply by marcelklehr** [reply]
I agree that yellow would be a good solution. I'll fix this soon, sorry for the wait!

**Comment by misterhsp** [comment]
Yes, yellow is a good idea.

**Comment by misterhsp** [comment]
I once tried to build it myself, but unfortunately failed.

...

**Reply by marcelklehr** [reply]
heh, yeah, it's become a bit of a beast

**Comment by misterhsp** [comment]
Just got the new update, but for me is no yellow icon shows up during the sync.

...

**Reply by marcelklehr** [reply]
Yeah, I tried yellow and it wasn't very visible :/ I opted to simplify the sync icon instead to a white circle, maybe you noticed. What do you think about it?

**Comment by misterhsp** [comment]
Try green, it should be more visible.

...
